extract_assets.py

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env python3
 
-import argparse, json, os, signal, time
-from multiprocessing import Pool, cpu_count, Event, Manager, ProcessError
+import argparse, json, os, signal, time, colorama, multiprocessing
```

Comment:
colorama is currently a diff.py requirement only (so optional like diff.py usage), I don't see a point making it a requirement for building

(you probably didn't realize it needs to be installed separately, it isn't in the standard python library)

Reply:
Then add the package as a dependency.
It isn't a big package and colored output is always nice

Reply:
Yes, I'd argue that we should encourage colour output in our Python scripts where appropriate. And Windows (as usual) doesn't support the ANSI stuff except in a proper terminal, which is what colorama is supposed to get around.

Reply:
then you should add instructions in the readme on installing it